test/dummy_pkg_ros2/main.cpp

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
```suggestion
// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
I think this should be updated to `2020` if its a new file.